src/icu4c.mk

```diff
@@ -20,35 +19,42 @@ define $(PKG)_UPDATE
 endef
 
 define $(PKG)_BUILD_COMMON
-    cd '$(1)/source' && autoreconf -fi
-    mkdir '$(1).native' && cd '$(1).native' && '$(1)/source/configure' \
+    cd '$(SOURCE_DIR)/source' && autoreconf -fi
+    mkdir '$(BUILD_DIR)/$(PKG).native'
+    cd '$(BUILD_DIR)/$(PKG).native' && '$(SOURCE_DIR)/source/configure' \
         CC=$(BUILD_CC) CXX=$(BUILD_CXX)
-    $(MAKE) -C '$(1).native' -j '$(JOBS)'
+        $(MAKE) -C '$(BUILD_DIR)/$(PKG).native' -j $(JOBS) VERBOSE=1
 
-    mkdir '$(1).cross' && cd '$(1).cross' && '$(1)/source/configure' \
+    mkdir '$(BUILD_DIR)/$(PKG).cross'
+    cd '$(BUILD_DIR)/$(PKG).cross' && '$(SOURCE_DIR)/source/configure' \
         $(MXE_CONFIGURE_OPTS) \
-        --with-cross-build='$(1).native' \
-        CFLAGS=-DU_USING_ICU_NAMESPACE=0 \
-        CXXFLAGS='--std=gnu++0x' \
-        SHELL=bash
-
-    $(MAKE) -C '$(1).cross' -j '$(JOBS)' install
+        --with-cross-build='$(BUILD_DIR)/$(PKG).native' \
+        CFLAGS='-DU_USING_ICU_NAMESPACE=0' \
+        CXXFLAGS='-std=c++11' \
+        LIBS='-lmsvcr100' \
+        SHELL=$(SHELL)
+    
+    rm -fv '$(PREFIX)/$(TARGET)/lib/libicu*.dll.a'
+    $(MAKE) -C '$(BUILD_DIR)/$(PKG).cross' -j $(JOBS) all-recursive VERBOSE=1 SHELL=$(SHELL)
+    $(MAKE) -C '$(BUILD_DIR)/$(PKG).cross' -j 1 install VERBOSE=1 SHELL=$(SHELL)
     ln -sf '$(PREFIX)/$(TARGET)/bin/icu-config' '$(PREFIX)/bin/$(TARGET)-icu-config'
 endef
 
 define $(PKG)_BUILD_SHARED
-    $($(PKG)_BUILD_COMMON)
+    $(call $(PKG)_BUILD_COMMON)
```

Comment:
Why add `call`?

Reply:
No reason, just cleaner I think.

Reply:
It's cleaner to only use `call` if you are actually sending parameters.